app/translate_survey.py

```diff
@@ -263,10 +263,12 @@ def translate_calculated_summary(container, translations):
         translate_titles_text(container, container['id'], translations)
         translate_titles_text(container['calculation'], container['id'], translations)
 
+
 def translate_definitions_text(container, context, translations):
     if 'definitions' in container:
-        for definitions in container['definitions']:
-            translate_container(definitions, context, translations)
+        for definition in container['definitions']:
+            translate_container(definition, context, translations)
+            translate_block_content(definition, context, translations)
```

Comment:
should this rename to `translate_content_block` maybe? as its not the `block` that this is now just used for?

Reply:
@jonnyshaw89 Sorted.